From 13f97532bf244a8433f2dffbb84783092f401b26 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jonas=20Dre=C3=9Fler?= <verdre@v0yd.nl>
Date: Mon, 9 Sep 2019 21:28:46 +0200
Subject: [PATCH 043/152] overviewControls: Remove slide transitions before
 setting value manually

Remove transitions of the `slide-x` property of the layout manager
before we set the property to a fixed value, otherwise the transitions
might still be running and change the value after we set it.

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/707
---
 js/ui/overviewControls.js | 1 +
 1 file changed, 1 insertion(+)

diff --git a/js/ui/overviewControls.js b/js/ui/overviewControls.js
index cf24a5e56..f5c51eed7 100644
--- a/js/ui/overviewControls.js
+++ b/js/ui/overviewControls.js
@@ -248,6 +248,7 @@ var SlidingControl = class {
         // selector; this means we can now safely set the full slide for
         // the next page, since slideIn or slideOut might have been called,
         // changing the visiblity
+        this.actor.remove_transition('@layout.slide-x');
         this.layout.slide_x = this._getSlide();
         this._updateTranslation();
     }
-- 
2.23.0

